### PR TITLE
Fix: Update java-managedkafka .repo-metadata.json

### PR DESCRIPTION
**Pull request by acocuzzo**
Fixes product name.

### CODE REVIEW

**Comment by noahdietz** [comment]
Hey @acocuzzo I don't work on this repo but was notified of the mismatched names via YAQS and was coming to do the same thing. There are a few spots that need updating, mostly in the various `pom.xml` files: https://github.com/search?q=repo%3Agoogleapis%2Fgoogle-cloud-java%20%22Google%20Apache%20Kafka%20for%20BigQuery%20API%22&type=code

Should those be done at the same time here?

**Comment by acocuzzo** [comment]
Fixed old mentions

File: generation_config.yaml [hunk 1]
```diff
@@ -1231,7 +1231,7 @@ libraries:
   - proto_path: google/cloud/managedidentities/v1beta1
 
 - api_shortname: managedkafka
-  name_pretty: Apache Kafka for BigQuery API
+  name_pretty: Managed Service for Apache Kafka API
```
Comment (suztomo) [inline generation_config.yaml:1234]: I believe this is the source of other lines.

**Review by suztomo — APPROVED** [review]

File: generation_config.yaml (hunk 1 above)
Comment (JoeWang1127) [reply generation_config.yaml:1234]: We shouldn't allow manual change in other files. I'll run the generation job for this library.

**Pull request merged**